### PR TITLE
spring cleaning (in autumn)

### PR DESCRIPTION
**Pull request by nylira**
* remove the relative Vue and Stylus links in favor of Webpack Resolve.Alias links
* fixed issues where css colors were undefined after switch to dark-theme
* fixed page navigation that wasn't converted to the new desktop-friendly system
* attempted to replace all 1px borders with variable (px = 0.0625rem) borders (for easier adjustment later)
* attempted to replace all fonts defined in pixels with variable font sizes (xs, sm, lg, xl)
* attempted to replace all magic color variables with standard colors
* remove old, commented out code
* removes PageReceive #103
* resolves loading bg not same as app bg #106 

This PR hangs on the the last `npm run test`, not sure what the issue is:

screen shot 2017-11-25 at 11 10 25 am

Also, I can't get gaia to run when I run `npm run testnet`. Is this an issue on develop too?

screen shot 2017-11-25 at 11 12 23 am

### CODE REVIEW

**Comment by nylira** [comment]
I've removed the nomination page for validators because nomination will be done in the CLI.

**Comment by nylira** [comment]
I'm not sure what changed over the weekend, but now all the tests are working and the gaia testnet is running fine. From my end, it looks like the PR is no longer causing any breaking issues.

File: app/src/renderer/components/common/NiFieldAddon.vue [hunk 1]
```diff
@@ -9,18 +9,18 @@ export default {
...
 .ni-field-addon + .ni-field
-  margin-left -1px
+  margin-left -1 * px
   position relative
   z-index 1
 
 .ni-field-addon
   background app-bg
-  border 1px solid bc
+  border px solid bc
```
Comment (faboweb) [inline app/src/renderer/components/common/NiFieldAddon.vue]: 0 here?

File: app/src/renderer/components/monitor/BlockchainSelectModal.vue [hunk 2]
```diff
@@ -46,7 +46,7 @@ export default {
   z-index 1000
 
 .modal-blockchain-container 
-  border-top 1px solid bc
+  border-top px solid bc
```
Comment (faboweb) [inline app/src/renderer/components/monitor/BlockchainSelectModal.vue]: is it maybe worth making border a mixin?

File: app/src/renderer/components/monitor/PageValidatorSlashes.vue [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+  div(slot="menu"): tool-bar
+    router-link(:to="{ name: 'validator', params: { validator: $route.params.validator }}")
       i.material-icons arrow_back
-    a(@click='toggleSearch'): i.material-icons search
+      .label Back
+    a(@click='toggleSearch')
+      i.material-icons search
+      .label Search
 
   part(title='Statistics')
```
Comment (faboweb) [inline app/src/renderer/components/monitor/PageValidatorSlashes.vue]: dd="2 weeks ago%" <- percent sign (not important right now)

Comment (faboweb) [reply app/src/renderer/components/monitor/PageValidatorSlashes.vue]: + list items link to nowhere

File: app/index.ejs [hunk 4]
```diff
@@ -29,7 +29,7 @@ body { margin: 0; }
   justify-content: center;
   width: 100vw;
   height: 100vh;
-  background: #000;
+  background: hsl(233, 36%, 13%);
```
Comment (faboweb) [inline app/index.ejs]: Would it be possible to make these colors relative to our stylesheet? Maybe build a css-file and then render it in at build-time of the index.html? (not high prio)

File: app/src/renderer/components/monitor/PageValidatorSlashes.vue [hunk 5]
```diff
@@ -1,10 +1,12 @@
 <template lang="pug">
 page(icon="storage" :title="`${validatorId} Slashes`")
-  tool-bar
-    router-link(
-      :to="{ name: 'validator', params: { validator: $route.params.validator }}")
+  div(slot="menu"): tool-bar
+    router-link(:to="{ name: 'validator', params: { validator: $route.params.validator }}")
```
Comment (faboweb) [inline app/src/renderer/components/monitor/PageValidatorSlashes.vue]: back-buttons do not work on validator sub-pages

File: app/src/renderer/components/monitor/PageBlockchain.vue [hunk 6]
```diff
@@ -1,11 +1,15 @@
 <template lang="pug">
```
Comment (faboweb) [inline app/src/renderer/components/monitor/PageBlockchain.vue]: I still can't see the page. It references state.blockchain that is not set anywhere (or I can't find it).

**Comment by faboweb** [comment]
Gaia has changed the API so we need to change the startup process. :/

**Comment by faboweb** [comment]
The user menu doesn't open anymore.

File: app/index.ejs (hunk 4 above)
Comment (nylira) [reply app/index.ejs]: Yes, we can do that later.

File: app/src/renderer/components/monitor/PageBlockchain.vue (hunk 6 above)
Comment (nylira) [reply app/src/renderer/components/monitor/PageBlockchain.vue]: This is resolved in a different PR #113

File: app/src/renderer/components/common/NiFieldAddon.vue (hunk 1 above)
Comment (nylira) [reply app/src/renderer/components/common/NiFieldAddon.vue]: Not zero, but I had `px` a variable for 1 pixel.

File: app/src/renderer/components/monitor/BlockchainSelectModal.vue (hunk 2 above)
Comment (nylira) [reply app/src/renderer/components/monitor/BlockchainSelectModal.vue]: Would be useful to do later!

File: app/src/renderer/components/monitor/PageValidatorSlashes.vue (hunk 3 above)
Comment (nylira) [reply app/src/renderer/components/monitor/PageValidatorSlashes.vue]: Fixed the % issue.

**Comment by nylira** [comment]
The user menu is resolved in another PR #115

**Comment by nylira** [comment]
I merged in `develop` so gaia is working again.

File: app/src/renderer/components/monitor/PageBlockchain.vue (hunk 6 above)
Comment (nylira) [reply app/src/renderer/components/monitor/PageBlockchain.vue]: The network monitor is on @mappum's plate. But #113 will show that the data is not loading yet.

**Pull request merged**